product_docs/docs/pgd/5/reference/pgd-settings.mdx

```diff
@@ -478,19 +478,21 @@ Track lock timing when tracking statistics for relations.
 
 ### `bdr.enable_wal_decoder`
 
-Enables Logical Change Record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this is false. When set to true, a decoding worker process starts and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.
+Enables logical change record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this setting is false. When set to true, a decoding worker process starts, and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.
 
-Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.
+!!! Note
+    You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to true on the group.
 
 ### `bdr.receive_lcr`
 
-Enables the node to request Logical Change Records are used when subscribed to another node. By default, this is false. When this setting is true on a downstream node, the node requests that upstream nodes use Logical Change Records (LCR) when sending to it.  If `bdr.enable_wal_decoder` is set to true on a node, this setting should also be `true`.
+Enables the node to request to use LCRs when subscribed to another node. By default, this setting is false. When this setting is true on a downstream node, the node requests that upstream nodes use LCRs when sending to it. If you set `bdr.enable_wal_decoder` to true on a node, also set this setting to `true`.
 
-Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.
+!!! Note 
+    You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to to true on the group.
 
 ### `bdr.lcr_cleanup_interval`
 
-Logical Change Record (LCR) file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worked stores LCR files as a buffer. These files are periodically cleaned and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.
+LCR file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worker stores LCR files as a buffer. These files are periodically cleaned, and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.
```

Comment:
This is a reference section and the reader is not reading the content sequentially. Keeping the full name and acronym (the explain the variable name) is reasonable. Run with the acronym first if needs be, but reference sections need to be relatively self-contained.

```suggestion
LCR (logical change record) file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worker stores LCR files as a buffer. These files are periodically cleaned, and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.
```

Reply:
Okay, but I'm going to switch them because the definition goes first and the initialism goes second.